components/hacbs/integration-service/overlays/kcp-unstable/apiexport.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: integration-service
+spec:
+  latestResourceSchemas:
+    - md5-bdbd79111419cb45ed5f46c858d8ff96.integrationtestscenarios.appstudio.redhat.com
+  permissionClaims:
+    - resource: pipelineruns
+      group: tekton.dev
+      identityHash: 72b2990e51b1931e9fee86e67091b721a8c32f407d762fc847d9d2316a988b52
+    - resource: applications
+      group: appstudio.redhat.com
+      identityHash: 835cb978eda43fb9776eab29818632aab365fedac2ee8e137440be5e5ed10e1d
+    - resource: components
+      group: appstudio.redhat.com
+      identityHash: 835cb978eda43fb9776eab29818632aab365fedac2ee8e137440be5e5ed10e1d
+    - resource: applicationsnapshots
+      group: appstudio.redhat.com
+      identityHash: 835cb978eda43fb9776eab29818632aab365fedac2ee8e137440be5e5ed10e1d
+    - resource: environments
+      group: appstudio.redhat.com
+      identityHash: 835cb978eda43fb9776eab29818632aab365fedac2ee8e137440be5e5ed10e1d
+    - resource: applicationsnapshotenvironmentbindings
+      group: appstudio.redhat.com
+      identityHash: 835cb978eda43fb9776eab29818632aab365fedac2ee8e137440be5e5ed10e1d
+    - resource: release
+      group: appstudio.redhat.com
+      identityHash: release-service
```

Comment:
Adding hashes in https://github.com/redhat-appstudio/infra-deployments/pull/824

Reply:
Thank you! I was thinking where to ask release-service identityHash, updated

Reply:
Yes, agree.